Upstream-Status: Backport

The version of texinfo in Debian Sarge does not understand the @euro{} command.
This patch replaces the @euro{} command with the word "euro".

--- gnutls-1.3.5/doc/gnutls.texi.orig	2006-04-26 08:06:40.918268000 +0930
+++ gnutls-1.3.5/doc/gnutls.texi	2006-04-26 08:06:52.446515440 +0930
@@ -11,8 +11,8 @@
 long as it is difficult enough to generate two different messages with
 the same hash algorithm output. In that case the same signature could
 be used as a proof for both messages. Nobody wants to sign an innocent
-message of donating 1 @euro{} to Greenpeace and find out that he
-donated 1.000.000 @euro{} to Bad Inc.
+message of donating 1 euro to Greenpeace and find out that he
+donated 1.000.000 euro to Bad Inc.
 
 For a hash algorithm to be called cryptographic the following three
 requirements must hold
